src/main/resources/templates/teams.html

```diff
@@ -112,11 +140,32 @@ <h2 class="team-heading team-heading-current-team" th:text="#{team.heading.publi
 	                               <td th:text="${teamEntry.value.description}">Description</td>
 	                               <!--<td th:text="${teamEntry.value.institution}">Institution</td>-->
 	                            </tr>
+                                <!--<tr>-->
+                                    <!--<td height="700"></td>-->
+                                <!--</tr>-->
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this commented out code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=Web%3AAvoidCommentedOutCodeCheck)